src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSpark.java

```diff
@@ -87,98 +99,186 @@ protected void runTool( final JavaSparkContext ctx ) {
         }
 
         final Locations locations =
-                new Locations(evidenceDir, intervalFile, qNamesMappedFile, kmerFile, qNamesAssemblyFile);
+                new Locations(evidenceDir, intervalFile, qNamesMappedFile,
+                                highFrequencyKmersFile, kmerFile, qNamesAssemblyFile);
         final PipelineOptions pipelineOptions = getAuthenticatedGCSOptions();
-        final SAMFileHeader header = getHeaderForReads();
-        final JavaRDD<GATKRead> reads = getUnfilteredReads();
-        final HopscotchHashSet<QNameAndInterval> qNamesSet;
-        final int nIntervals;
-        {
-            final HopscotchHashSet<KmerAndInterval> kmerAndIntervalsSet;
-            {
-                final ReadMetadata readMetadata = getMetadata(header, getMeanBasesPerTemplate(reads));
-                log("Metadata retrieved.");
-                final Broadcast<ReadMetadata> broadcastMetadata = ctx.broadcast(readMetadata);
-                final List<Interval> intervals =
-                        getIntervals(broadcastMetadata, header, reads, locations, pipelineOptions);
-
-                nIntervals = intervals.size();
-                log("Discovered " + nIntervals + " intervals.");
-
-                if ( nIntervals == 0 ) return;
-
-                qNamesSet = getQNames(ctx, broadcastMetadata, intervals, reads);
-                broadcastMetadata.destroy();
-
-                final int initialSetSize = qNamesSet.size();
-                log("Discovered " + initialSetSize + " mapped template names.");
-
-                // remove intervals that have ridiculously high coverage -- these are mapper artifacts
-                cleanQNames(qNamesSet, intervals, readMetadata.getMeanBasesPerTemplate(), locations, pipelineOptions);
-                log("Killed " + (initialSetSize-qNamesSet.size()) + " intervals that had >" + MAX_COVERAGE + "x coverage.");
+        final JavaRDD<GATKRead> unfilteredReads = getUnfilteredReads();
+        final JavaRDD<GATKRead> allPrimaryLines =
+                unfilteredReads.filter(read -> !read.isSecondaryAlignment() && !read.isSupplementaryAlignment());
+        final int[] nIntervals = new int[1];
```

Comment:
This is not a huge deal but returning the number of intervals in an array of size 1 is stylistically not a very Java-like thing to do. Could you return a results object or maybe just a `Tuple2<Integer, HopscotchHashSet<QNameAndInterval>>`? Nitpicky, I know, but more in keeping with Java style.